File: MD/SP/ScriptsAndServices/HomePageEnabled/ScriptAndServiceMenuItem.vb
Public Class ScriptAndServiceMenuItem
    Inherits System.Windows.Forms.ToolStripMenuItem

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(ByRef tdrData As System.Data.DataRow, ByVal Text As String)
        MyBase.New(Text)
        drData = tdrData
    End Sub

    Private drData As System.Data.DataRow
    Public Property gsData() As System.Data.DataRow
        Get
            Return drData
        End Get
        Set(ByVal value As System.Data.DataRow)
            drData = value
        End Set
    End Property


End Class

File: M/MD/MauiDUDEForCustomerService/MauiDUDE/UserInterface/HomePage/GenericComponents/ScriptAndServiceMenuItem.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace MauiDUDE
{
    public class ScriptAndServiceMenuItem : ToolStripMenuItem
    {
        public DataRow gsData { get; set; }

        public ScriptAndServiceMenuItem() : base()
        {

        }

        public ScriptAndServiceMenuItem(DataRow row, string text) : base(text)
        {
            gsData = row;
        }
    }
}
